feature/src/main/java/org/woowatechcamp/banchan/feature/custom/CartBottomSheetFragment.kt

```diff
@@ -0,0 +1,70 @@
+package org.woowatechcamp.banchan.feature.custom
+
+import android.graphics.Paint
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
+import org.woowatechcamp.banchan.feature.R
+import org.woowatechcamp.banchan.feature.databinding.CartBottomSheetContentBinding
+
+class CartBottomSheetFragment : BottomSheetDialogFragment() {
+    private var _binding: CartBottomSheetContentBinding? = null
+    private val binding get() = requireNotNull(_binding)
+    private val countFlow = MutableStateFlow(1)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = DataBindingUtil.inflate(
+            layoutInflater,
+            R.layout.cart_bottom_sheet_content,
+            container,
+            false
+        )
+
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initViews()
+        setObserver()
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
+    private fun initViews() {
+        with(binding) {
+            tvCartBottomSheetNPrice.paintFlags = Paint.STRIKE_THRU_TEXT_FLAG
+            btnCartBottomSheetCancel.setOnClickListener { this@CartBottomSheetFragment.dismiss() }
+            btnCartBottomSheetMinus.setOnClickListener { if (countFlow.value > 1) countFlow.value-- }
+            btnCartBottomSheetPlus.setOnClickListener { if (countFlow.value < 99) countFlow.value++ }
+        }
+    }
+
+    private fun setObserver() {
+        viewLifecycleOwner.lifecycleScope.launch {
+            viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
+                countFlow.collectLatest { binding.count = it }
+            }
```

Comment:
`androidx.lifecycle`에서 제공하는 FlowExt에 있는 `flowWithLifecycle`을 활용하면 좀 더 간결하게 표현가능합니다!
```suggestion
            countFlow.flowWithLifecycle(viewLifecycleOwner.lifecycle) {
            binding.count = it
            }
```

Reply:
조언 감사합니다~ 반영하겠습니다.